react-app/components/ChonkyComponent/folderSearch.js

```diff
@@ -0,0 +1,56 @@
+const folderSearch = (data1, folderChainTemp, currentFolder) => {
+  console.log("search", data1, folderChainTemp, currentFolder);
+
+  let filesTemp = [];
+  for (let i = 0; i < data1.length; i++) {
```

Comment:
can we use `index` instead of `i`?

Reply:
I will remove this component, I'm not using it